app/controllers/provider_foreman_controller.rb

```diff
@@ -724,13 +711,7 @@ def replace_right_cell(replace_trees = [])
     trees = {}
     trees[:configuration_manager_providers] = build_configuration_manager_tree(:configuration_manager_providers, :configuration_manager_providers_tree) if replace_trees
 
-    # Build presenter to render the JS command for the tree update
-    presenter = ExplorerPresenter.new(
-      :active_tree => x_active_tree,
-      :delete_node => @delete_node,      # Remove a new node from the tree
-    )
-    r = proc { |opts| render_to_string(opts) }
-
+    presenter, r = rendering_objects
```

Comment:
in the future, I'd do this one in a separate PR since it's not really related to rubocop fixes.  I had to git grep to realize this method does exactly what the prior code was doing.